showcase/src/assets/translations/en.json

```diff
@@ -432,6 +432,49 @@
     "NEWS": {
       "TITLE": "News"
     },
+    "NGX_FORM_ERRORS": {
+      "TITLE": "Reactive Forms errors",
+      "EXAMPLE": "Example",
+      "FORM": {
+        "VALIDATION": {
+          "REQUIRED": "{{fieldName}} is required",
+          "PASSWORD_REQUIRED": "{{fieldName}} must be provided",
+          "USER_NAME": {
+            "UNIQUE": "This username has already been taken"
+          },
+          "PASSWORD": {
+            "MAX_LENGTH": "Password cannot be more than {{requiredLength}} characters long",
+            "MIN_LENGTH": "Password must be at least {{requiredLength}} characters long",
+            "PATTERN": "The password must contain at least one uppercase, one lowercase, and one number"
```

Comment:
I would keep the sentence structure similar here. Eg: "Password must contain at least one uppercase, one lowercase, and one number"